test/test_other.py

```diff
@@ -10226,8 +10226,7 @@ def test_separate_dwarf(self):
         # with name sections but no code sections.
         # if sec.type == webassembly.SecType.CODE:
         #   self.fail(f'section of type "{sec.type}" found in separate dwarf file')
-        if sec.name and sec.name != 'name' and not sec.name.startswith('.debug'):
-          self.fail(f'non-debug section "{sec.name}" found in separate dwarf file')
+        pass
```

Comment:
oops, this is now very ugly. I just reopened #13084 again. If you'd prefer we can just delete this whole for loop until it gets fixed.

Reply:
I'm ok with it is as is. There is a TODO so it's clear the code is meant to be fixed up later.